homeassistant/components/template/config_flow.py

```diff
@@ -160,32 +160,43 @@ def _validate_unit(options: dict[str, Any]) -> None:
         and (units := DEVICE_CLASS_UNITS.get(device_class)) is not None
         and (unit := options.get(CONF_UNIT_OF_MEASUREMENT)) not in units
     ):
-        units_string = sorted(
-            [str(unit) if unit else "no unit of measurement" for unit in units],
+        sorted_units = sorted(
+            [f"'{str(unit)}'" if unit else "no unit of measurement" for unit in units],
```

Comment:
No need for a f-string here. It's a single value `str(unit)`. I believe we should revert this patch.

Reply:
That's not the case, the string is now quoted.
Before this change: `UnitOfTime.MICROSECONDS -> µs`
With this change: `UnitOfTime.MICROSECONDS -> 'µs'`

Reply:
Oh! Sorry, you are absolutely right. Nvmnd then 👍